src/types/augments/tinyMCE.d.ts

```diff
@@ -28,7 +28,7 @@ declare global {
     type EditorSelection = _tinymceTypes.EditorSelection;
     type Entities = _tinymceTypes.Entities;
     type Env = _tinymceTypes.Env;
-    type EventDispatcher<T> = _tinymceTypes.EventDispatcher<T>;
+    type EventDispatcher<T extends {}> = _tinymceTypes.EventDispatcher<T>;
```

Comment:
This is the initial problem that made this file start failing. This change was probably introduced back when Typescript formalised the definition of `{}` a bit more so that it's less janky.